modules/retail/pages/retail-install-setup.adoc

```diff
@@ -1,42 +1,69 @@
 [[retail-install-setup]]
 = Setting up
 
-When you have installed the [guimenu]``SUSE Manager for Retail Server`` and the [guimenu]``SUSE Manager Retail Branch Server``, you need to set up your environment by following these steps:
+This section requires an installed and configured {susemgr} Server, one or more {smr} branch servers, and one or more {susemgr} build host.
 
-. Install the build host and register it to {susemgr}
-. Configure the build host
-. Create the required system group
+Next steps in configuring {smr} environment are:
 
-These are the branch server setup steps:
+. Prepare POS images
+. Configure {smr} branch server services
+. Synchronize POS images to {smr} branch servers
 
-// already done:
-// . Install the branch server and register it to {susemgr}
-. Assign and configure branch server formulas
-. Synchronize images to the branch servers
+These steps are usually performed multiple times in lifetime of {smr} and are loosely related.
 
-Each procedure is detailed in this section.
+Typically POS images are periodically rebuild when updated packages are available and then synchronized to the {smr} branch servers before update window.
 
+Configuration of {smr} branch server services have to be done each time a new branch server is registered.
 
+== Prepare and Build Terminal Images
 
-== Create Required System Groups
+For information about {susemgr} image building, see xref:administration:image-management.adoc[].
 
-{smr} requires system groups for terminals and servers.
-Manually create these system groups during installation:
+{smr} POS images are images specifically tailored for {smr} environment and designed to be deployed using PXE booting mechanism.
 
-* [systemitem]``TERMINALS``
-* [systemitem]``SERVERS``
+=== POS Image Templates
 
-Additionally, you will need to create a system group for each branch server, and each terminal hardware type in your environment.
+As starting point, {suse} provides basic templates at https://github.com/SUSE/manager-build-profiles/tree/master/OSImage[].
+These templates need to be adapted for specific usecases, for example by including specific applications, configuration settings, and users.
 
-You can create system groups using the {susemgr} {webui}.
-Navigate to menu:Systems[System Groups] and click btn:[Create System Group].
+[IMPORTANT]
+====
+POS templates do not describe any system user. Using {suse} provided templates as is will result in system without possibility of user login. Salt management of the system is unaffected, however.
```

Comment:
```suggestion
By default, POS templates do not include a system user. 
You will not be able to login as a user to a system that has been installed with a {suse} provided template. 
This does not apply to Salt-managed terminals.
```

This admon needs a remedy.

Reply:
> This does not apply to Salt-managed terminals.

This is not correct. What I meant to say is that the salt management of the terminals is still possible and it is unrelated to missing user configuration. In other words, it is also possible to add user using salt after the terminal was deployed.

Reply:
Better now?